search-commons/src/test/resources/resource_urls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "query=Metoder+for+måle+vannføring+i+umålte+vassdrag": 0,
```

Comment:
This is not good test. I think we want a test that is named something like:

`shouldReturnTicketsUserHasAccessRightsToViewWhenSearchingForAllTickets()`

and 

`shouldNotReturnTicketsUserHasNoAccessRightsToViewWhenSearchingForAllTickets()`
 
and similar test for aggregations.

Reply:
Added this test to prove that the error in test/dev, doesn't have anything to do with the code.....
https://sikt.atlassian.net/browse/NP-48140